src/main/kotlin/app/revanced/patches/music/layout/permanentshuffle/patch/PermanentShufflePatch.kt

```diff
@@ -0,0 +1,40 @@
+package app.revanced.patches.music.layout.permanentshuffle.patch
+
+import app.revanced.extensions.toErrorResult
+import app.revanced.patcher.annotation.Description
+import app.revanced.patcher.annotation.Name
+import app.revanced.patcher.data.BytecodeContext
+import app.revanced.patcher.extensions.InstructionExtensions.addInstructionsWithLabels
+import app.revanced.patcher.extensions.InstructionExtensions.getInstruction
+import app.revanced.patcher.patch.BytecodePatch
+import app.revanced.patcher.patch.PatchResult
+import app.revanced.patcher.patch.PatchResultSuccess
+import app.revanced.patcher.patch.annotations.Patch
+import app.revanced.patches.music.annotations.MusicCompatibility
+import app.revanced.patches.music.layout.permanentshuffle.fingerprints.PermanentShuffleFingerprint
+import app.revanced.patcher.util.smali.ExternalLabel
+
+@Patch
```

Comment:
Is this patch something, that most people would like to have enabled? If not use `@Patch(false)`